libshortfin/src/shortfin/support/iree_helpers.h

```diff
@@ -11,6 +11,7 @@
 #include <stdexcept>
 
 #include "iree/base/api.h"
+#include "iree/base/internal/file_io.h"
```

Comment:
fyi: file_io.h sucks and will be removed at some point (why it's in internal/) - fine for now and when it does get axed or changed you can copy it local if you still want it

Reply:
ack. It is better than what was done in the prior bindings. Will take the guts off your hands when needed.